pilz_teleoperation/README.md

```diff
@@ -111,4 +111,10 @@ pick_pose = Pose(
       w = 0.0
     )
 )
+```
+
+### Display Stored Points
+Stored points can be displayed as TFs. To do this run rviz activate the tf module and then run:
```

Comment:
To "run rviz" maybe reference the launch command on top of the readme? That has a TF view activated already
```
roslaunch prbt_jog_arm_support prbt_jog_arm_sim.launch
```
Maybe move the "bringup" section one paragraph up out of the "Keyboard Teleoperation" section?

Reply:
Id say its on the right spot, in the package its implemented, next to store points.
pilz_teloperation package is not prbt specific, so i would not refer to it here.